lib/rmt.rb

```diff
@@ -1,5 +1,5 @@
 module RMT
-  VERSION ||= '2.5.8'.freeze
+  VERSION ||= '2.5.10'.freeze
```

Comment:
@jesusbv The version bump is `2.5.9`.
```suggestion
  VERSION ||= '2.5.9'.freeze
```